### PR TITLE
RavenDB-17353 - `GetNodeTagById` method return redundant whitespace at the beginning of a node tag

### PR DESCRIPTION
**Pull request by shiranshalom**
### Issue link

https://issues.hibernatingrhinos.com/issue/RavenDB-17353

### Type of change

- Bug fix

### How risky is the change?

- Low 

### Backward compatibility

- Non breaking change

### Is it platform specific issue?

- No

### Documentation update

- No documentation update is needed 

### Testing 

- It has been verified by manual testing

### Is there any existing behavior change of other features due to this change?

- No

### UI work

- No UI work is needed

### CODE REVIEW

**Review by ayende — COMMENTED** [review]
What is going on here?
Please add a testcase

**Comment by shiranshalom** [comment]
> What is going on here? Please add a testcase
@ayende 
https://issues.hibernatingrhinos.com/issue/RavenDB-17353#focus=Comments-67-124381.0-0

**Comment by ppekrol** [comment]
Let's write a unit test then. Since this is a static method, you do not need to setup raven, replication, etc. Should be fairly straightforward to do.

File: test/SlowTests/Issues/RavenDB_17353.cs [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+        [Fact]
+        public void CheckGetNodeByTag()
+        {
+            const string dbid1 = "07e2GrSMdkunq1AC+KgwIg";
+            const string dbid2 = "F9I6Egqwm0Kz+K0oFVIR9Q";
+            const string cv = "C:8397-07e2GrSMdkunq1AC+KgwIg, A:8917-3UiZOcXaZ0+d6GI/VTr//A, B:8397-5FYpkl5TX0SPlIBPwjmhUw, A:2568-F9I6Egqwm0Kz+K0oFVIR9Q, A:13366-IG4VwBTOnkqoT/uwgm2OQg, A:2568-OSKWIRBEDEGoAxbEIiFJeQ";
+            
+            // without the change:
+            var nodeTag = GetNodeTagById(cv, dbid1, includeChanges: false);
```
Comment (ppekrol) [inline test/SlowTests/Issues/RavenDB_17353.cs]: This is incorrect. You need to use `ChangeVectorUtils.GetNodeTagById` method, you do not need the one that you wrote. The purposed of e.g. TDD is to write a failing test, fix the issue, and then the test should pass.

Comment (shiranshalom) [reply test/SlowTests/Issues/RavenDB_17353.cs]: Oh, sorry. Done

File: test/SlowTests/Issues/RavenDB_17353.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            const string dbid1 = "07e2GrSMdkunq1AC+KgwIg";
+            const string dbid2 = "F9I6Egqwm0Kz+K0oFVIR9Q";
+            const string cv = "C:8397-07e2GrSMdkunq1AC+KgwIg, A:8917-3UiZOcXaZ0+d6GI/VTr//A, B:8397-5FYpkl5TX0SPlIBPwjmhUw, A:2568-F9I6Egqwm0Kz+K0oFVIR9Q, A:13366-IG4VwBTOnkqoT/uwgm2OQg, A:2568-OSKWIRBEDEGoAxbEIiFJeQ";
+            
+            var nodeTag = ChangeVectorUtils.GetNodeTagById(cv, dbid1);
+            Assert.Equal("C", nodeTag);
+
+            nodeTag = ChangeVectorUtils.GetNodeTagById(cv, dbid2);
+            Assert.NotEqual(" A", nodeTag); 
+            Assert.Equal("A", nodeTag);
```
Comment (ppekrol) [inline test/SlowTests/Issues/RavenDB_17353.cs:27]: small note, previous assertion is not needed, mind that next time

**Review by ppekrol — APPROVED** [review]

**Pull request merged**